ethcore/vm/src/action_params.rs

```diff
@@ -96,11 +96,11 @@ impl Default for ActionParams {
 	/// Returns default ActionParams initialized with zeros
 	fn default() -> ActionParams {
 		ActionParams {
-			code_address: Address::new(),
+			code_address: Address::default(),
```

Comment:
The PR seems to replace `H256::default` with `H256::zero`, what's the reasoning for using `default` here?